accepted/color-4-new-spaces.md

```diff
@@ -1055,28 +1059,27 @@ The individual conversion algorithms are:
 
 ### Gamut Mapping
 
-> Some [known color space]s describe limited color gamuts. If a color is 'out of
-> gamut' for a particular space (most often because of conversion from a
-> larger-gamut color-space), it can be useful to 'map' that color to the nearest
-> available 'in-gamut' color. Gamut mapping is the process of finding an
-> in-gamut color with the least objectionable change in visual appearance.
-
-Gamut mapping color `origin`, a [known color space] `destination`, and an
-unquoted string `method`. It returns a color in `origin`'s color space.
+> Some [known color space]s describe limited color gamuts. If a color is "out of
+> gamut" for a particular space (most often because of conversion from a
+> larger-gamut color-space), it can be useful to "map" that color to the nearest
+> available "in-gamut" color. Gamut mapping is the process of finding an
+> in-gamut color with the least objectionable change in visual appearance, which
+> is to some degree inherently contextual and subjective.
 
-* Let `origin-space` be `origin`'s color space.
-
-* If either `origin-space` or `destination` is not a [known color space], throw
-  an error.
+Gamut mapping takes a color `origin`, a [known color space] `destination`, and
+an unquoted string `method`. It returns a color in `origin`'s color space.
 
 * Let `color` be the result of [converting] `origin` into `destination`.
 
-* If `method` is not one of the methods defined below, throw an error.
+* If `method` is not case-insensitively equal to one of the methods defined
+  below, throw an error.
+
+* If `color.is-in-gamut(color)`, return `origin`.
 
 * Let `mapped` be the result of running the method defined below whose name
-  matches `method`. If no such method matches, throw an error.
+  matches `method`.
 
-* Return the result of [converting] `mapped` into `origin-space`.
+* Return the result of [converting] `mapped` into `origin`'s space.
 
 #### `local-minde`
```

Comment:
should the spec in `local-minde` specifically refer to the 13 February 2024 draft of CSS Color Module Level 4 ? What happens if the algorithm in the CSS spec is updated between that draft and the CR of Color Level 4 ?